doc/osh-help.md

```diff
@@ -55,13 +55,18 @@ Otherwise:
 - `bin/oil` runs `~/.config/oil/oilrc`
 
 Pass --rcfile /dev/null to disable this behavior.
+Pass --norc to disable both --rcfile and --rcdir.
 
-If the --rcdir flag is specified, all files in that folder will be executed on startup.
-Othewise:
+If the --rcdir flag is specified, files in that folder will be executed on
+startup.
+Otherwise:
 
 - `bin/osh` runs everything in `~/.config/oil/oshrc.d/`
 - `bin/oil` runs everything in `~/.config/oil/oilrc.d/`
 
+Pass --rcdir /dev/nullto disable this behavior.
```

Comment:
Let's remove this line about /dev/null -- this isn't really a good pattern, since /dev/null is a file, not a directory.  `--norc` is the better way of doing it

I think it's sufficient to document `--norc` once, or say 

- "Pass --rcfile /dev/null or --norc to disable the startup file"
- "Pass --norc to disable the startup directory"

That way it's not redundant